pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,17 +16,51 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repositories` A multi-valued structure with the following child members:
+* `repository` The name of the repository. It is recommended not to include spaces in the repository name.
+* `variants` A multi-valued structure containing the `name, `path`, and `canonical` values.
+The `canonical` tags tell the search engine that of this group of pages with similar content, one of them is "official" and we want it to be considered the most relevant.
```

Comment:
You shouldn't use "quotes" to define a term. And it begs the question: What does "official" mean?

Is it the default version?

How do you generate a variant? Maybe explain that in the example: How to publish the canonical version and how to publish a variant.